andela/iceman-backend
# chore: setting up babel and dotenv

**What does this PR do?**

This PR is for setting babel and dotenv

  **Description of Task to be completed?**

 - installed required babel modules
 - installed dotenv
- imported dotenv in the index.js module
- created server under scripts to test the configuration
- I set the node targets version to v10 and above in the babel configuration to add support for async and await.

  **How should this be manually tested?**

  - You can confirm if babel is trans-piling your code correctly by running npm run server 
  - You can set up any environmental variable in .env file from the project root directory, then you have access to that environment variable by simply using process.env.[NAME OF THE ENVIRONMENTAL VARIABLE]

 **Any background context you want to provide?**

  xxx

 **What are the relevant stories?**

  #167735027 - Setup dotenv 

 **Screenshots (if appropriate)**

  xxx

 **Questions:**

 xxx
